src/vivarium/framework/lookup/table.py

```diff
@@ -200,11 +218,11 @@ class CategoricalTable(LookupTable):
     def __init__(
         self,
         table_number: int,
-        data: Union[ScalarValue, pd.DataFrame, List[ScalarValue], Tuple[ScalarValue]],
-        population_view_builder: Callable,
-        key_columns: Union[List[str], Tuple[str]],
-        value_columns: Union[List[str], Tuple[str]],
-        **kwargs,
+        data: pd.DataFrame,
+        population_view_builder: Callable[..., PopulationView],
```

Comment:
This doesn't seem restrictive enough? We're calling this a few lines later, and it is taking a list of strings.